core/contracts/financial-templates/implementation/Liquidatable.sol

```diff
@@ -26,30 +26,53 @@ contract Liquidatable is PricelessPositionManager {
     using SafeMath for uint;
     using SafeERC20 for IERC20;
 
+    /****************************************
+     *     LIQUIDATION DATA STRUCTURES      *
+     ****************************************/
+
     // Because of the check in withdrawable(), the order of these enum values should not change.
     enum Status { Uninitialized, PreDispute, PendingDispute, DisputeSucceeded, DisputeFailed }
 
     struct LiquidationData {
-        /** Following variables set upon creation of liquidation */
+        // -> Following variables set upon creation of liquidation
         address sponsor; // Address of the liquidated position's sponsor
         address liquidator; // Address who created this liquidation
         Status state; // Liquidated (and expired or not), Pending a Dispute, or Dispute has resolved
         uint liquidationTime; // Time when liquidation is initiated, needed to get price from Oracle
-        /** Following variables determined by the position that is being liquidated */
+        // -> Following variables determined by the position that is being liquidated
         FixedPoint.Unsigned tokensOutstanding; // Synthetic Tokens required to be burned by liquidator to initiate dispute
         FixedPoint.Unsigned lockedCollateral; // Collateral locked by contract and released upon expiry or post-dispute
         // Amount of collateral being liquidated, which could be different from
         // lockedCollateral if there were pending withdrawals at the time of liquidation
         FixedPoint.Unsigned liquidatedCollateral;
         // Unit value (starts at 1) that is used to track the fees per unit of collateral over the course of the liquidation.
         FixedPoint.Unsigned rawUnitCollateral;
-        /** Set upon initiation of a dispute */
+        // -> Following variable set upon initiation of a dispute
```

Comment:
nit: why the arrow here but not elsewhere?

Reply:
these are sections comments, not for lines. wanted to find a way to denote these sections in a consistent way. What do you suggest for this?